Orange/widgets/visualize/owmosaic.py

```diff
@@ -359,21 +356,12 @@ def __init__(self):
             callback=self.set_color_data, model=self.color_model)
         self.bar_button = gui.checkBox(
             box2, self, 'use_boxes', label='Compare with total',
-            callback=self._compare_with_total)
+            callback=self.update_graph)
         gui.rubber(self.controlArea)
 
     def sizeHint(self):
         return QSize(720, 530)
 
-    def _compare_with_total(self):
-        if self.data is not None and \
-                self.data.domain.class_var is not None and \
-                self.interior_coloring != self.CLASS_DISTRIBUTION:
-            self.interior_coloring = self.CLASS_DISTRIBUTION
-            self.coloring_changed()  # This also calls self.update_graph
-        else:
-            self.update_graph()
-
```

Comment:
If I haven't overlooked something, this method was called by a combo that was disabled when `self.interior_coloring` was not `self.CLASS_DISTRIBUTION`, so the condition was never `True`.